kong/plugins/opentelemetry/schema.lua

```diff
@@ -46,10 +47,27 @@ return {
         { resource_attributes = resource_attributes },
         { batch_span_count = { type = "integer", required = true, default = 200 } },
         { batch_flush_delay = { type = "integer", required = true, default = 3 } },
+        { queue = typedefs.queue },
```

Comment:
Should `batch_span_count` and `batch_flush_delay` be modified to simply `{ type = "integer"}` like for the other plugins, or this is an exception?